<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/icon/icon/write.png</normaloff>:/new/icon/icon/write.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_O"/>
   <addaction name="action_S"/>
   <addaction name="action_Z"/>
   <addaction name="action_R"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_Z"/>
    <addaction name="action_R"/>
    <addaction name="separator"/>
    <addaction name="action_A"/>
    <addaction name="action_X"/>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="action_F5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="action_2"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_N">
   <property name="icon">
    <iconset theme="new" resource="icon.qrc">
     <normaloff>:/new/icon/icon/new.png</normaloff>:/new/icon/icon/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/icon/open.png</normaloff>:/new/icon/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/icon/save.png</normaloff>:/new/icon/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>设置背景颜色</string>
   </property>
  </action>
  <action name="action_P">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/icon/undo.png</normaloff>:/new/icon/icon/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_L">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_F3">
   <property name="text">
    <string>查找下一个(F3)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>替换(&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_F5">
   <property name="text">
    <string>日期/时间(F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
  <action name="action_MyNotepad">
   <property name="text">
    <string>关于MyNotepad</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="action_R">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/icon/redo.png</normaloff>:/new/icon/icon/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
